extractor/main_test.go

```diff
@@ -0,0 +1,412 @@
+package main
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"github.com/gnolang/gno/gno.land/pkg/sdk/vm"
+	"github.com/gnolang/gno/tm2/pkg/amino"
+	"github.com/gnolang/gno/tm2/pkg/crypto"
+	"github.com/gnolang/gno/tm2/pkg/sdk/bank"
+	"github.com/gnolang/gno/tm2/pkg/std"
+	"github.com/go-test/deep"
+	"github.com/stretchr/testify/require"
+	"math/rand"
+	"os"
+	"path/filepath"
+	"sort"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+)
+
+const (
+	numSourceFiles    = 20
+	numTx             = 100
+	numMsg            = 200
+	maxFilesPerPkg    = 100
+	maxFileBodyLength = 200
+	msgPerTx          = numMsg / numTx
+	txPerSourceFile   = numTx / numSourceFiles
+	maxDepositAmount  = 5000
+	maxArgs           = 2
+	sourceFileType    = ".log"
+)
```

Comment:
My idea was to have a config for the tests, all editable in one spot. However, I moved the constants that are not used more than once into the scopes there are used in. 

[20dd043](https://github.com/gnolang/tx-exports/pull/5/commits/20dd04324dda8cd17481efe188e7a458e214e493)

Reply:
You should localize these values within each individual test, because you introduce unnecessary coupling by keeping them as constants.

99% of these can be dropped if you simplify your mocks, as mentioned in https://github.com/gnolang/tx-exports/pull/5#discussion_r1427815777, since there is no need for the complexity

Reply:
Resolved as part of other conversations